src/main/java/seedu/dengue/model/predicate/FindPredicate.java

```diff
@@ -45,8 +45,24 @@ public FindPredicate(Optional<Name> name, Optional<SubPostal> subPostal, Optiona
         this.dateRange = dateRange;
         this.ageRange = ageRange;
     }
+
+    private boolean isEveryPrefixEmpty(Optional<Name> name, Optional<SubPostal> subPostal,
```

Comment:
Maybe name it something like `allPrefixesEmpty`? So it sounds more like a boolean condition than a question.

Reply:
![image](https://user-images.githubusercontent.com/100463724/230729478-51ca38e5-8dfe-4567-be4e-89f29c030dad.png)
Should be can to leave it as is for a boolean method